.husky/pre-commit

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+. "$(dirname "$0")/_/husky.sh"
+
+npx lint-staged
```

Comment:
Fair warning, this takes the latest version I believe which means you may hit some drift issues if they update it in a breaking way.

Reply:
Yeah seems like too many problems here. Also no file suppression comment tsc can add?? I'll probably give up on this one